### PR TITLE
[featur/apof-8] Added restaturan managing funcionality.

### PR DESCRIPTION
**Pull request by kamilsokolowski**

### CODE REVIEW

File: apof/apof/settings/common.py [hunk 1]
```diff
@@ -37,6 +37,8 @@
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
+    'apof',
+    'order',
```
Comment (Orzech150) [inline apof/apof/settings/common.py]: Do we need another app to manage restaurants?

File: apof/order/models.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	"""
+
+	name = models.CharField('nazwa restauracji', max_length=255)
+	address = models.CharField('adres restauracji', max_length=255)
+	phone_number = models.CharField('numer telefonu', max_length=16)
+	order_time_start =  models.TimeField('od godziny:')
+	order_time_end =  models.TimeField('do godziny:')
+
+	def __str__(self):
+		return str(self.name)
```
Comment (Orzech150) [inline apof/order/models.py]: Well, `self.name` is CharField so you don't need `str()` method.

File: apof/apof/settings/common.py (hunk 1 above)
Comment (pkucmus) [inline apof/apof/settings/common.py]: Yeap, I'd leave it to be simply `apof`.

File: apof/order/models.py (hunk 2 above)
Comment (pkucmus) [inline apof/order/models.py]: @Orzech150 is right. Additionally use 4 spaces not tabs.

**Comment by pkucmus** [comment]
Closing due to inactivity.

**Pull request closed without merging**